### PR TITLE
[ironic] Disable virtual media for now on non-UEFI nodes

### PR DESCRIPTION
**Pull request by fwiesel**

### CODE REVIEW

**Review by sandzwerg — APPROVED** [review]

**Comment by sandzwerg** [comment]
I merged before waiting for a second review, that was not intended

**Comment by BerndKue** [comment]
I realized that when I wanted to give my approval.

**Pull request merged**
